src/transformers/training_args.py

```diff
@@ -866,12 +890,39 @@ class TrainingArguments:
             ),
         },
     )
+    fsdp_config: Optional[str] = field(
+        default=None,
+        metadata={
+            "help": (
+                "Config to be used with fsdp (Pytorch Distributed Parallel Training). The  value is either the of "
+                "deepspeed json config file (e.g., `ds_config.json`) or an already loaded  json file as `dict`."
+            )
+        },
+    )
+    fsdp_backward_prefetch: str = field(
+        default="backward_pre",
+        metadata={
+            "help": (
+                "Which  `backward_prefetch` mode to use out of 'backward_pre' or 'backward post' when using PyTorch"
+                " Fully Sharded Data Parallel (FSDP) training (in distributed training only)."
+            ),
+        },
+    )
+    fsdp_forward_prefetch: Optional[bool] = field(
+        default=False,
+        metadata={
+            "help": (
+                "If True, then FSDP explicitly prefetches the next upcoming all-gather"
+                "while executing in the forward pass"
+            ),
+        },
+    )
```

Comment:
```suggestion
```

Reply:
You haven't removed these fields. These aren't required now as they are handled in the `fsdp_config`

Reply:
This still needs to be addressed.